root/static/scripts/common/artworkViewer.js

```diff
@@ -147,8 +147,14 @@ $.widget("mb.artworkViewer", $.ui.dialog, {
 
         this._sizeAndPosition();
 
-        this.element.find("img").remove().end().append(image);
-        this.$loading.stop(true, true).fadeOut();
+        this.element
+            .find("img")
+            .remove()
+            .end()
+            .append(image);
+        this.$loading
+            .stop(true, true)
+            .fadeOut();
```

Comment:
This looked fine how it was. I would only break them across multiple lines if there are more than 3 method calls in the chain and/or the line doesn't fit within 78 characters.